### PR TITLE
Set minHeight based on fontSize or style.

### PR DESCRIPTION
**Pull request by SergioEstevao**
## Description
Make sure we have initial min-height that works properly for all platforms.

## How has this been tested?
This can be tested on this mobile GB PR:https://github.com/wordpress-mobile/gutenberg-mobile/pull/723

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate.

### CODE REVIEW

File: packages/editor/src/components/rich-text/index.native.js [hunk 1]
```diff
@@ -520,7 +520,7 @@ export class RichText extends Component {
 
 			this.lastEventCount = undefined; // force a refresh on the native side
 		}
-		let minHeight = 0;
+		let minHeight = this.props.fontSize ? this.props.fontSize : styles[ 'editor-rich-text' ].minHeight;
```
Comment (hypest) [inline packages/editor/src/components/rich-text/index.native.js]: 👋 @SergioEstevao , I think the fontSize value is too small as an initial value for the Android side. If you see the minHeights previously used (https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/src/variables.android.scss#L9-L11) you'll notice that the values are greater than 50. A quick test I did with this current implementation shows a value of `24` in fontSize.

This way, the glitch is still noticeable in the title block for example.

File: packages/editor/src/components/rich-text/index.native.js [hunk 2]
```diff
@@ -520,7 +520,7 @@ export class RichText extends Component {
 
 			this.lastEventCount = undefined; // force a refresh on the native side
 		}
-		let minHeight = 0;
+		let minHeight = styles[ 'editor-rich-text' ].minHeight;
```
Comment (hypest) [inline packages/editor/src/components/rich-text/index.native.js]: There's a check for the validity of `style` and `style.minHeight` right after this line. Can we consolidate them somehow?

Comment (etoledom) [reply packages/editor/src/components/rich-text/index.native.js]:
```javascript
let minHeight = styles[ 'editor-rich-text' ].minHeight;
if ( style && style.minHeight ) {
	minHeight = style.minHeight;
}
```

`style` comes from `props`. This seems to be a good way to let the user of RichText modify the default `minHeight`.

Comment (etoledom) [reply packages/editor/src/components/rich-text/index.native.js]: @hypest did you mean something else by `consolidate` them?
We could set `minHeight` at once but the code might be less readable?

Something like:
```javascript
const minHeight = (style && style.minHeight) || styles[ 'editor-rich-text' ].minHeight;
```

What do you think @marecar3 ?

Comment (marecar3) [reply packages/editor/src/components/rich-text/index.native.js]: @etoledom looking good to me :)

**Comment by marecar3** [comment]
Let's merge it @etoledom.

**Comment by etoledom** [comment]
Thank you!

File: packages/editor/src/components/rich-text/index.native.js (hunk 2 above)
Comment (hypest) [reply packages/editor/src/components/rich-text/index.native.js]: Late to reply but, want to verify that yeah, that looks good to me too!

**Pull request merged**